f8a_worker/dispatcher/nodes.yml

```diff
@@ -500,6 +500,15 @@
         encryption: false
         versioned: true
 
+    - name: 'S3KronosAppend'
+      import: 'f8a_worker.storages'
+      configuration:
+        <<: *configurationS3
+        bucket_name: 'dev-stack-analysis-clean-data'
```

Comment:
Wouldn't be better to prefix the bucket with deployment prefix so different deployments don't share the same bucket? Take a look for example here: https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/493/files#diff-7847c95658ce3aa270db29af1f9a2aabR471

Reply:
Currently we don't have a prefix for this bucket.

Reply:
We will definitely need to have a separate bucket for different deployments in future. Since we want to merge this to master and possibly promote it to prod in the near future, wouldn't be better to prefix the bucket now?

Reply:
Ok, we can do that, and have the parameter take no value.

Reply:
`bucket_name: '{DEPLOYMENT_PREFIX}-stack-analysis-clean-data'` would always point to a deployement-specific bucket. You don't have to worry about the DEPLOYMENT_PREFIX, it will be automatically substituted for the correct value. We already use this trick on other places.

Reply:
And just to clarify: correct value means "prod" for prod, "STAGE" for stage, "<your-dev-cluster-username>" for your dev cluster deployment and so on.